msr-flows/msbr_compare/run_user_input.py

```diff
@@ -448,70 +450,46 @@ def cycle_time_decay(self, identifier='CTD'):
                     label=SP_identifier,
                     alpha=ui.overlap,
                     lw=ui.width)
-            for each_step in range(ui.number_serp_steps):
```

Comment:
Why did you remove this for loop?

Reply:
Good question! I had noticed having that large outer loop was making things less organized, so I moved the loop into the `serpent_plotting` script functions if I decide to use the loop. Line 60 of that file contains this same line, but it made things more organized doing it this way I think.